cli/src/main.rs

```diff
@@ -463,6 +482,53 @@ async fn run_cache(command: &CacheCommand) -> Result<(), CliError> {
     Ok(())
 }
 
+async fn run_init(args: &InitArgs) -> Result<(), CliError> {
+    let template_json = include_str!("templates.json");
+    let mappings: HashMap<String, Vec<String>> = serde_json::from_str(template_json)?;
+    let language = args.language.to_lowercase().trim().to_owned();
+
+    let Some(url) = mappings
+        .iter()
+        .find_map(|(k, v)| v.contains(&language).then_some(k)) else {
```

Comment:
You probably won't need the string owned
```suggestion
    let language = args.language.to_lowercase().trim();

    let Some(url) = mappings
        .iter()
        .find_map(|(k, v)| v.contains(language).then_some(k)) else {
```